picker/src/main/java/me/angrybyte/numberpicker/view/ActualNumberPicker.java

```diff
@@ -66,8 +66,8 @@
     private Paint mBarPaint;
     private RectF mBarBounds = new RectF(0, 0, 0, 0);
     private int mBarCount = DEFAULT_BAR_COUNT;
-    private int mMinBarWidth = 1;
```

Comment:
I think this change is unnecessary as it controls the vertical bar width and doesn't affect the X movement or the stored value.